lookout/style/format/analyzer.py

```diff
@@ -553,3 +560,19 @@ def _load_config(cls, config: Mapping[str, Any]) -> Mapping[str, Any]:
                 raise ValueError("Config %s can not be merged with default values config: "
                                  "%s: %s" % (config, global_config, e)) from None
         return effective_config
+
+    @classmethod
+    def _check_language_version(cls, language: str, data_service: DataService,
+                                log: logging.Logger) -> bool:
+        """
+        Return the value indicating whether the Babelfish driver version for the specified \
+        language is supported.
+        """
+        language = language.lower()
+        try:
+            data_service.check_bblfsh_driver_versions([
+                language + import_module("lookout.style.format.langs.%s" % language).__version__])
```

Comment:
```suggestion
                "%s==%s" % (language, import_module("lookout.style.format.langs.%s" % language).__version__]))
```

Reply:
Nope. It supports any valid version specifier compatible with PEP (don't remember the number, you can google yourself).